src/main/java/com/teammetallurgy/atum/entity/EntityDesertWolf.java

```diff
@@ -480,30 +418,33 @@ public boolean canMateWith(EntityAnimal par1EntityAnimal) {
         } else if (!(par1EntityAnimal instanceof EntityDesertWolf)) {
             return false;
         } else {
-            EntityDesertWolf entitywolf = (EntityDesertWolf) par1EntityAnimal;
-            return !entitywolf.isTamed() ? false : (entitywolf.isSitting() ? false : this.isInLove() && entitywolf.isInLove());
+            EntityDesertWolf entityDesertWolf = (EntityDesertWolf) par1EntityAnimal;
+            return !entityDesertWolf.isTamed() ? false : (entityDesertWolf.isSitting() ? false : this.isInLove() && entityDesertWolf.isInLove());
         }
     }
 
-    public boolean func_70922_bv() {
-        return this.dataWatcher.getWatchableObjectByte(19) == 1;
+    public int getCollarColor() {
+        return this.dataWatcher.getWatchableObjectByte(20) & 15;
+    }
+
+    public void setCollarColor(int collarColor) {
+        this.dataWatcher.updateObject(20, Byte.valueOf((byte) (collarColor & 15)));
     }
 
     @Override
     public EntityAgeable createChild(EntityAgeable par1EntityAgeable) {
-        EntityDesertWolf entitywolf = new EntityDesertWolf(this.worldObj);
+        EntityDesertWolf entityDesertWolf = new EntityDesertWolf(this.worldObj);
         String s = this.func_152113_b();
 
         if (s != null && s.trim().length() > 0) {
-            entitywolf.func_152115_b(s);
-            entitywolf.setTamed(true);
+            entityDesertWolf.func_152115_b(s);
+            entityDesertWolf.setTamed(true);
         }
 
-        return entitywolf;
+        return entityDesertWolf;
     }
 
-    @Override
-    public EntityLivingBase getOwner() {
-        return null;
+    public boolean func_70922_bv() {
+        return this.dataWatcher.getWatchableObjectByte(19) == 1;
```

Comment:
Kindly revert moving of the function,
Old location: https://github.com/TeamMetallurgy/Atum/pull/148/files#diff-3a33804d239ffb942f87589ddf58ab6dL488